src/SecurityInsights/examples/Get-AzSentinelAutomationRule.md

```diff
@@ -0,0 +1,30 @@
+### Example 1: List all Automation Rules
+```powershell
+PS C:\> Get-AzSentinelAutomationRule -ResourceGroupName "myResourceGroupName" -workspaceName "myWorkspaceName"
+
+DisplayName                 : VIP automation rule
```

Comment:
Plus, please split your example code and output with following pattern
\```powershell
{code}
\```
\```output
{code}
\```

Reply:
fixed in latest commit